docs/usage.md

```diff
@@ -194,6 +196,23 @@ Please note the following requirements:
 2. The metadata file has to follow the [QIIME2 specifications](https://docs.qiime2.org/2019.10/tutorials/metadata/)
 3. In case of multiple sequencing runs, specific naming of samples are required, see [here](#--multipleSequencingRuns)
 
+The first column in the metadata file is the identifier (ID) column and defines the sample or feature IDs associated with your study.
+Additional columns defining metadata associated with each sample or feature ID are optional.
+Metadata files are not required to have additional metadata columns, so a file containing only an ID column is a valid QIIME 2 metadata file.
+It is not recommended to mix sample and feature IDs in a single metadata file; keep sample and feature metadata stored in separate files.
+Identifiers should be 36 characters long or less, and also contain only ASCII alphanumeric characters
+(i.e. in the range of [a-z], [A-Z], or [0-9]), the period (.) character, or the dash (-) character.
+By default all numeric columns, blanks or NA are removed, and only columns with multiple different values but not all unique are selected.
+The columns which are to be assessed can be specified by `--metadata_category`, see below.
```

Comment:
Well, if `--metadata_category` isn't specified than all columns that fit the specification are automatically chosen. Maybe would be better to mention that here as well?